src/server/shell/default_configuration.cpp

```diff
@@ -194,6 +195,8 @@ auto mir::DefaultServerConfiguration::the_accessibility_manager() -> std::shared
                 the_input_event_transformer(),
                 the_options()->get<bool>(mir::options::enable_key_repeat_opt),
                 the_cursor(),
-                std::make_shared<shell::BasicMouseKeysTransformer>(the_main_loop(), the_clock()));
+                std::make_shared<shell::BasicMouseKeysTransformer>(the_main_loop(), the_clock()),
+                std::make_shared<shell::BasicSimulatedSecondaryClickTransformer>(
+                    the_main_loop(), the_input_event_transformer()->virtual_device_id()));
```

Comment:
This is fragile: `the_input_event_transformer()` returns a temporary, and the only guarantee that the lifetime of the input event transformer that returns `virtual_device_id()` extends beyond the statement is that there's another `the_input_event_transformer()` temporary being passed to `BasicAccessibilityManager`